pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```diff
@@ -63,10 +65,12 @@ let
       cxxStandard = "17";
     };
   };
+
```

Comment:
Maybe we can add a comment here:
`Aarch64 default gcc9 will cause ICE. For reference #108305`